### PR TITLE
Changing utility name of u-text-capitalise

### LINKED ISSUE #119: Change `u-text-capitalise` namespace to `u-text-transform`

### PR DESCRIPTION
**Pull request by chris-pearce**
This closes #119.

### CODE REVIEW

**Comment by AlecRaeside** [comment]
LGTM

File: utilities/_u-text.scss [hunk 1]
```diff
@@ -312,46 +312,46 @@ $u-text-breakpoint-toggle-hyphenate:              false !default;
...
 }// end if
 
-%u-text-capitalise-capitalise,
-.u-text-capitalise-capitalise {text-transform: capitalize;}
+%u-text-transform-capitalise,
+.u-text-transform-capitalise {text-transform: capitalize;}
 
 @if $u-text-breakpoint-toggle-capitalise-capitalise {
-  @include generate-at-breakpoints('.u-text-capitalise-capitalise',
+  @include generate-at-breakpoints('.u-text-transform-capitalise',
```
Comment (kllevin) [inline utilities/_u-text.scss:338]: :+1: so much better!

**Comment by kllevin** [comment]
You need to update all the breakpoint toggle names!

**Comment by chris-pearce** [comment]
@kllevin @AlecRaeside this will break your stuff

**Comment by kllevin** [comment]
LGTM

**Pull request merged**